docs/project/requirements/architecture.md

```diff
@@ -0,0 +1,147 @@
+# Architecture
+
+Un schéma d'implémentation détaillé sera établi pour décrire l'architecture globale de l'infrastructure, y compris les différents composants, leur interaction et leur déploiement.
+
+
+## Application
+
+
+### Proxy: Traefik
+
+* Application router
+* High-availability by fault tolerance
+* Load-balancing by requests distribution
+  ALB ?
```

Comment:
EKS uses ELB.

Reply:
Yes, but which type? Application or Network LB?

Reply:
I had to look into this. We're using a NLB, and we're actually setting it in our config for traefik:
`service.beta.kubernetes.io/aws-load-balancer-type: nlb`

Reply:
OK, thanks.
So we have the ELB's NLB doing L3/L4 load balancing on our Traefik ALB doing L7 route balancing, right?

Reply:
Correct.